src/chocolatey/infrastructure.app/services/INugetService.cs

```diff
@@ -16,9 +16,11 @@
 
 namespace chocolatey.infrastructure.app.services
 {
-    using configuration;
+	using chocolatey.infrastructure.results;
+	using configuration;
+	using System.Collections.Concurrent;
 
-    public interface INugetService : ISourceRunner
+	public interface INugetService : ISourceRunner

```

Comment:
Looks like there is a whitespace issue here, where the main definition is bumped over an extra tab.